NDB.Covid19/NDB.Covid19.Droid/Utils/ForegroundServiceHelper.cs

```diff
@@ -0,0 +1,41 @@
+﻿using Android.App;
+using Android.Content;
+using Android.OS;
+using Android.Runtime;
+using Android.Views;
+using Android.Widget;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+
+namespace NDB.Covid19.Droid.Utils
+{
+    public class ForegroundServiceHelper
+    {
+        public static void StartForegroundServiceCompat<T>(Context context, Bundle args = null) where T : Service
+        {
+            Intent intent = new Intent(context, typeof(T));
```

Comment:
Add try/catch block with general exception here. And in the catch block log the exception